lib/language_syntax/golang_syntax.dart

```diff
@@ -0,0 +1,57 @@
+// Обработка правильности отступов, точек с запятой для цикла for
+// с учетом комментариев, с учетом строк.
+Map<int, String> findGolangErrors(String text) {
+  List<String> lines = text.split("\n");
+  Map<int, String> errors = {};
+  int indentLevel = 0;
+
+  for (int i = 0; i < lines.length; i++) {
+    String currentLine = lines[i];
+
+    if (currentLine.trim().isEmpty ||
+        currentLine.startsWith(RegExp("\\s*//"))) {
+      continue;
+    }
+
+    if (currentLine.startsWith(RegExp("\\s*/\\*"))) {
+      while ((!currentLine.contains(RegExp("\\*/\\s*"))) &&
+          (i < lines.length - 1)) {
+        i++;
+        currentLine = lines[i];
+      }
+    } else if (currentLine.startsWith(RegExp(".*`"))) {
+      do {
+        i++;
+        currentLine = lines[i];
+      } while ((!currentLine.contains(RegExp("`"))) && (i < lines.length - 1));
+    }
+
+    if (currentLine.trim().endsWith("}")) {
+      indentLevel--;
+    }
+
+    for (int countOfSpace = 0;
+        countOfSpace < currentLine.length;
+        countOfSpace++) {
```

Comment:
I think that the design for should be all in one line, if it does not go beyond the screen size.

Reply:
I don't do that, but the code reformer built into android studio.

Reply:
I see that there's exactly similar line in `java_dart_syntax.dart`, that wasn't formatted like that. And I do not see any reason for dividing that expression into several lines